pubnub-kotlin/pubnub-kotlin-core-api/src/commonMain/kotlin/com/pubnub/api/PubNubError.kt

```diff
@@ -239,6 +239,12 @@ enum class PubNubError(private val code: Int, val message: String) {
         181,
         "Channel and/or ChannelGroup contains empty string which is not allowed.",
     ),
+
+    HERE_NOW_OFFSET_OUT_OF_RANGE(
+        182,
+        "HereNow offset is out of range. Valid range is 0 to infinity.",
```

Comment:
Currently we know that up to the `1000`. There definitely will be an upper limit, but it can't be `infinity`.